src/templates/base.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html lang="{% block lang %}{{ language.lang_attribute }}{% endblock %}">
+{% block htmltag %}<html lang="{{ language.lang_attribute }}">{% endblock %}
```

Comment:
Very minor but I'd recommend doing this in a way that doesn't interfere with the lang attribute.
```suggestion
<html lang="{% block lang %}{{ language.lang_attribute }}{% endblock %}" {% block dir_attribute %}{% endblock %}>
```

And in the Arabic template you can specify:

```html
{% block dir_attribute %}dir="rtl"{% endblock %}
```

So it's unset by default but overridable in child templates.

Reply:
I'm in two minds about this.

I did try to go similar to what you did but ended up changing to what I submitted for a few reasons:

- It's not a `dir_attribute`, it's an `html_tag_other_attributes`, which got a little long-winded.
- I quite liked the ability to override the tag completely, so you could specify the order for example. Don't think it really matters here, but [examples online](https://www.w3.org/International/questions/qa-html-dir) have `dir` before `lang` for example.
- It could be in future certain languages might want to override the language, or include the region as well as the language (e.g. if we have an en-GB).

Anyway don't think it's a big deal whichever way we go but if we are going your way then think we should change the block name from `dir_attribute` but surely there's a better name than `html_tag_other_attributes`!!

Reply:
Any thoughts or preferences here @mikegeyser?

If not will just go with what @rviscomi suggests some we can close this out.

Reply:
I really like @rviscomi's suggestion.

Reply:
Done! Decision made.